cpp/include/cudf/io/json.hpp

```diff
@@ -640,6 +760,76 @@ class json_reader_options_builder {
     return *this;
   }
 
+  /**
+   * @brief Set whether json validation should be strict or not.
+   *
+   * @param val Boolean value to indicate whether json validation should be strict or not.
+   * @return this for chaining
+   */
+  json_reader_options_builder& strict_validation(bool val)
+  {
+    options.set_strict_validation(val);
+    return *this;
+  }
+
+  /**
+   * @brief Set Whether leading zeros are allowed in numeric values. strict validation must
+   * be enabled for this to have any effect.
+   *
+   * @note: strict_validation must be enabled for this to work.
```

Comment:
Since we throw in `json_reader_options`, we can do similar here too.
(Please also update docs)

Reply:
updated.